WordPress/Classes/Utility/PushNotificationsManager.swift

```diff
@@ -186,9 +186,6 @@ final public class PushNotificationsManager: NSObject {
             return
         }
 
-        // Analytics
-        trackNotification(with: userInfo)
-
```

Comment:
Removing this line from `PushNotificationsManager.handleNotification` resolved the issue of duplicated `push_notification_alert_tapped` analytics events. The idea is that `handleNotification` should only be concerned with handling the notification, irrespective of the context that triggered it (whether the notification was received in the background, or the notification alert was tapped).